squad_client/commands/create_or_update_project.py

```diff
@@ -62,6 +62,16 @@ def register(self, subparser):
             "--important-metadata-keys",
             help="Important metadata keys separated by comma",
         )
+        parser.add_argument(
+            "--build-confidence-count",
+            help="",
```

Comment:
Could you add the help text for this one?

Reply:
Done!